### PR TITLE
[feat-5116] create detail page for standaardteksten

### PR DESCRIPTION
**Pull request by SireeKoolenWijkstra**
Context:
On this page a user can change the details of the standard text he has clicked on in the overviewpage. In contrast to the design there is validation on subcategories, title and the text. Reuse of components causes error messages to appear above the component. 

Not included:
-  the page where a user can select multiple categories
- functionality for when a user wants to create a new standard text -> will be dealt with in a separate story

Ticket: [SIG-5116](https://gemeente-amsterdam.atlassian.net/browse/SIG-5116)

## Signalen

Before opening a pull request, please ensure:

- Make sure your PR title follows naming conventions: [feat-1234]: name feature
- Double-check your branch is based on `main` and targets `main`
- Pull request has tests (we are going for 100% coverage!)
- Code is well-commented, linted and follows project conventions
- Committed source code is headed by the correct SPDX license expression

Be kind to code reviewers, please try to keep pull requests as small and focused as possible :)

### CODE REVIEW

File: src/signals/incident-management/containers/DefaultTextsAdmin/new_DefaultTextsAdmin/components/DefaulTextForm/DefaulTextForm.tsx [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+  // onChange: (key: string, value: Value) => void
+  onChange: () => void
+}
+
+export const DefaultTextsForm: FC<DefaultTextsFormProps> = ({
+  value,
+  onChange,
+}: DefaultTextsFormProps) => {
+  return (
+    <>
```
Comment (OscarBakker) [inline src/signals/incident-management/containers/DefaultTextsAdmin/new_DefaultTextsAdmin/components/DefaulTextForm/DefaulTextForm.tsx:21]: not needed

File: src/signals/incident-management/containers/DefaultTextsAdmin/new_DefaultTextsAdmin/components/DefaulTextForm/DefaulTextForm.tsx [hunk 2]
```diff
@@ -0,0 +1,72 @@
...
+import { StyledButton, StyledColumn, StyledLabel } from '../../styled'
+
+type Value = { title: string; text: string; is_active: boolean }
+type DefaultTextsFormProps = {
+  value: Value
+  // onChange: (key: string, value: Value) => void
+  onChange: () => void
+}
+
+export const DefaultTextsForm: FC<DefaultTextsFormProps> = ({
```
Comment (OscarBakker) [inline src/signals/incident-management/containers/DefaultTextsAdmin/new_DefaultTextsAdmin/components/DefaulTextForm/DefaulTextForm.tsx:16]: remove FC

File: src/signals/incident-management/containers/DefaultTextsAdmin/new_DefaultTextsAdmin/components/SelectForm/SelectForm.tsx [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+  defaultTextsOptionList: Option[]
+  onFetchDefaultTexts: () => void
+}
+
+export const SelectForm = ({
+  defaultTextsOptionList,
+  onFetchDefaultTexts,
+}: SelectFormProps) => {
+  return (
+    <>
```
Comment (OscarBakker) [inline src/signals/incident-management/containers/DefaultTextsAdmin/new_DefaultTextsAdmin/components/SelectForm/SelectForm.tsx:18]: remove empy <>

File: src/signals/incident-management/containers/DefaultTextsAdmin/new_DefaultTextsAdmin/pages/Detail.tsx [hunk 4]
```diff
@@ -0,0 +1,52 @@
...
+import { FormContainer, StyledFormFooter } from '../styled'
+
+export const Detail = () => {
+  const title = 'Standaardtekst wijzigen'
+  const redirectURL = '..'
+
+  const genericFunction = () => {}
+
+  return (
+    <>
```
Comment (OscarBakker) [inline src/signals/incident-management/containers/DefaultTextsAdmin/new_DefaultTextsAdmin/pages/Detail.tsx:18]: remove

File: src/signals/incident-management/containers/DefaultTextsAdmin/new_DefaultTextsAdmin/styled.ts [hunk 5]
```diff
@@ -0,0 +1,70 @@
+// SPDX-License-Identifier: MPL-2.0
```
Comment (OscarBakker) [inline src/signals/incident-management/containers/DefaultTextsAdmin/new_DefaultTextsAdmin/styled.ts:1]: put styles in component folders

File: src/signals/incident-management/containers/DefaultTextsAdmin/new_DefaultTextsAdmin/components/PageHeader/PageHeader.tsx [hunk 6]
```diff
@@ -0,0 +1,39 @@
...
+import {
+  StyledHeading,
+  StyledHeadingWrapper,
+  StyledSection,
+} from '../../styled'
+
+type Props = {
+  title: string
+  children?: ReactNode
+  BackLink?: ReactNode
```
[the remark this answers is not in the document]
Comment (SireeKoolenWijkstra) [reply src/signals/incident-management/containers/DefaultTextsAdmin/new_DefaultTextsAdmin/components/PageHeader/PageHeader.tsx:16]: Made lowercase

File: src/signals/incident-management/containers/DefaultTextsAdmin/new_DefaultTextsAdmin/components/DefaulTextForm/DefaulTextForm.tsx (hunk 2 above)
Comment (SireeKoolenWijkstra) [reply src/signals/incident-management/containers/DefaultTextsAdmin/new_DefaultTextsAdmin/components/DefaulTextForm/DefaulTextForm.tsx:16]: removed

File: src/signals/incident-management/containers/DefaultTextsAdmin/new_DefaultTextsAdmin/components/DefaulTextForm/DefaulTextForm.tsx (hunk 1 above)
Comment (SireeKoolenWijkstra) [reply src/signals/incident-management/containers/DefaultTextsAdmin/new_DefaultTextsAdmin/components/DefaulTextForm/DefaulTextForm.tsx:21]: removed

File: src/signals/incident-management/containers/DefaultTextsAdmin/new_DefaultTextsAdmin/components/PageHeader/PageHeader.tsx [hunk 7]
```diff
@@ -0,0 +1,39 @@
...
+  filter?: {
+    name: string
+    options: any
```
[the remark this answers is not in the document]
Comment (SireeKoolenWijkstra) [reply src/signals/incident-management/containers/DefaultTextsAdmin/new_DefaultTextsAdmin/components/PageHeader/PageHeader.tsx:19]: set it to Option

File: src/signals/incident-management/containers/DefaultTextsAdmin/new_DefaultTextsAdmin/components/SelectForm/SelectForm.tsx (hunk 3 above)
Comment (SireeKoolenWijkstra) [reply src/signals/incident-management/containers/DefaultTextsAdmin/new_DefaultTextsAdmin/components/SelectForm/SelectForm.tsx:18]: removed it

File: src/signals/incident-management/containers/DefaultTextsAdmin/new_DefaultTextsAdmin/styled.ts (hunk 5 above)
Comment (SireeKoolenWijkstra) [reply src/signals/incident-management/containers/DefaultTextsAdmin/new_DefaultTextsAdmin/styled.ts:1]: Done

**Review by vdegraaf — COMMENTED** [review]
Tried to test, but does not work yet. 

Functional
- Infinite loop to fetch details —> it fetches with id 4 by default. 
    - You need to fetch the ID form the url, not as prop
- “Terug naar overzicht” does not work (check with Oscar’s busticket)
- An option for “alles” exist, but does not exist in the designs, is this correct?
- I’d expect to return to the overview page when I click “Verwijderen”
- The red alert bar is not showing when I click on save with an error. (Might be overkill though)
- Not able to change or save a standard text

Styling
- “Selecteer subcategory” button is out of proportions
- The radio buttons missing a header
- The input fields are not according to design
- Footer is not aligned correctly

**Comment by vdegraaf** [comment]
Functional
- If I delete a texts, I return to the overview page. However, the DELETE request is cancelled and the overview still shows the texts. However, on refresh it is deleted.
- Same behaviour when editing a text
- Setting the text to active/inactive is not saved
- The status of a text does not correspond with what is set in the Django admin. It is always “gemeld”
- If I click on the “Tekst toevoegen” button on the overview page I am not redirected to the detail page. 

Styling
- In overview page on hover I expect a pointer

File: src/shared/services/configuration/endpoint-definitions.js [hunk 8]
```diff
@@ -95,6 +95,8 @@ export default {
   // for reference.
   STANDARD_TEXTS_SEARCH_ENDPOINT: '/signals/v1/private/status-messages/search',
   // Private. Used by the incident management module to retrieve and search standard texts.
+  STANDARD_TEXTS_DETAILS_ENDPOINT: '/signals/v1/private/status-messages/',
+  // Private. Used by the incident management module to retrieve the details of a standard text.
```
Comment (vdegraaf) [inline src/shared/services/configuration/endpoint-definitions.js:99]: It is to retrieve data, but also patch, post and delete.

File: src/signals/incident-management/containers/StandardTextsAdmin/components/DetailPage/Detail.tsx [hunk 9]
```diff
@@ -0,0 +1,281 @@
...
+  StyledTextArea,
+} from './styled'
+import type { StandardTextForm, StandardTextDetailData } from './types'
+import { createPatch } from './utils'
+import { showGlobalNotification } from '../../../../../../containers/App/actions'
+import {
+  TYPE_LOCAL,
+  VARIANT_ERROR,
+} from '../../../../../../containers/Notification/constants'
+import { getErrorMessage } from '../../../../../../shared/services/api/api'
```
Comment (vdegraaf) [inline src/signals/incident-management/containers/StandardTextsAdmin/components/DetailPage/Detail.tsx:40]: You can remove the ../../ here

File: src/signals/incident-management/containers/StandardTextsAdmin/components/DetailPage/utils.ts [hunk 10]
```diff
@@ -0,0 +1,17 @@
+// SPDX-License-Identifier: MPL-2.0
+// Copyright (C) 2023 Gemeente Amsterdam
+
+import type { StandardTextForm, StandardTextDetailData } from './types'
+
+export const createPatch = (
+  data: StandardTextForm,
+  dirtyFields: any
```
Comment (vdegraaf) [inline src/signals/incident-management/containers/StandardTextsAdmin/components/DetailPage/utils.ts:8]: Add type

File: src/signals/incident-management/containers/StandardTextsAdmin/components/SelectedSubcategories/SelectedSubcategories.tsx [hunk 11]
```diff
@@ -0,0 +1,49 @@
...
+// Copyright (C) 2023 Gemeente Amsterdam
+import { useMemo } from 'react'
+
+import type { FieldError } from 'react-hook-form'
+import { useSelector } from 'react-redux'
+
+import Button from 'components/Button'
+import { makeSelectSubCategories } from 'models/categories/selectors'
+
+import { StyledErrorMessage } from '../../../../../my-incidents/components/LoginForm/styled'
```
Comment (vdegraaf) [inline src/signals/incident-management/containers/StandardTextsAdmin/components/SelectedSubcategories/SelectedSubcategories.tsx:11]: Ideally, if this component is shared, it should be brought to a shared level. However, in this case it is just some padding, which you could also just add to the styled.ts here.

File: src/signals/incident-management/containers/StandardTextsAdmin/components/SelectedSubcategories/mock-subcategories.ts [hunk 12]
```diff
@@ -0,0 +1,382 @@
+export const mockSubcategory = [
+  {
+    public_name: 'Afval',
```
Comment (vdegraaf) [inline src/signals/incident-management/containers/StandardTextsAdmin/components/SelectedSubcategories/mock-subcategories.ts:3]: Better to put in a __test__ folder, as we do on other places too.

File: src/signals/incident-management/containers/StandardTextsAdmin/components/Summary/Summary.tsx [hunk 13]
```diff
@@ -13,17 +13,18 @@ import statusList from '../../../../definitions/statusList'
...
-export const Summary = ({ standardText }: Props) => {
-  const { state, title, text } = standardText
+export const Summary = ({ standardText, onClick }: Props) => {
+  const { state, title, text, id } = standardText
 
   const status = statusList.find(({ key }) => key === state) as StatusType
 
   return (
-    <Wrapper>
+    <Wrapper onClick={() => onClick(id)} data-testid="summary-standard-text">
```
Comment (vdegraaf) [inline src/signals/incident-management/containers/StandardTextsAdmin/components/Summary/Summary.tsx:27]: The testid is not used.

File: src/signals/incident-management/index.js [hunk 14]
```diff
@@ -106,10 +111,10 @@ const IncidentManagement = () => {
...
           )}
           {configuration.featureFlags.showStandardTextAdminV2 && (
-            <Route
-              path={routes.standardTexts}
-              element={<StandardTextsAdmin />}
-            />
+            <Route path={routes.standardTexts}>
+              <Route index element={<StandardTextsAdmin />} />
+              <Route path={`:id`} element={<StandardTextsDetail />} />
+            </Route>
```
Comment (vdegraaf) [inline src/signals/incident-management/index.js:117]: Not sure if this is the correct way to implement this. @OscarBakker it seems with IncidentDetail and IncidentOverview it works differently. Can you check this?

**Review by vdegraaf — COMMENTED** [review]
Styling:
- I'd expect a pointer on the overview page, because it is clickable

File: src/signals/incident-management/containers/StandardTextsAdmin/components/DetailPage/Detail.test.tsx [hunk 15]
```diff
@@ -0,0 +1,183 @@
...
+  categories: [79, 4],
+  created_at: '24-06-2023',
+  state: 'o',
+  text: 'Mooie omschrijving',
+  title: 'Mooie titel',
+  updated_at: '25-06-2023',
+}
+
+describe('Detail', () => {
+  beforeEach(() => {
```
Comment (vdegraaf) [inline src/signals/incident-management/containers/StandardTextsAdmin/components/DetailPage/Detail.test.tsx:46]: Use adequate spacing for readability.

File: internals/testing/msw-server.ts [hunk 16]
```diff
@@ -233,6 +233,10 @@ const handlers = [
     res(ctx.status(200), ctx.json(incidentFixture))
   ),
 
+  rest.patch(API.STANDARD_TEXTS_DETAILS_ENDPOINT, (_req, res, ctx) =>
```
Comment (vdegraaf) [inline internals/testing/msw-server.ts:236]: you're using two ways of mocking now. The GET inside the test, and then PATCH and DEL inside this MSW Server. Might be good to choose one.

File: internals/testing/msw-server.ts [hunk 17]
```diff
@@ -233,6 +233,10 @@ const handlers = [
...
+    res(ctx.status(200), ctx.json(null))
```
Comment (vdegraaf) [inline internals/testing/msw-server.ts:237]: When you patch, you do get a response

File: internals/testing/msw-server.ts [hunk 18]
```diff
@@ -254,6 +258,11 @@ const handlers = [
     res(ctx.status(200))
   ),
 
+  //DELETE
+  rest.delete(API.STANDARD_TEXTS_DETAILS_ENDPOINT, (_req, res, ctx) =>
+    res(ctx.status(200), ctx.json(null))
+  ),
```
Comment (vdegraaf) [inline internals/testing/msw-server.ts:264]: hen you delete, the status is 204.

File: src/signals/incident-management/containers/StandardTextsAdmin/components/DetailPage/Detail.test.tsx [hunk 19]
```diff
@@ -0,0 +1,183 @@
...
+import type { StandardTextDetailData } from './types'
+import * as API from '../../../../../../../internals/testing/api'
+import {
+  fetchMock,
+  mockRequestHandler,
+} from '../../../../../../../internals/testing/msw-server'
+import {
+  TYPE_LOCAL,
+  VARIANT_ERROR,
+} from '../../../../../../containers/Notification/constants'
```
Comment (vdegraaf) [inline src/signals/incident-management/containers/StandardTextsAdmin/components/DetailPage/Detail.test.tsx:20]: remove ../../

**Review by vdegraaf — COMMENTED** [review]
Looking good! Just left some small comments

File: src/signals/incident-management/containers/DefaultTextsAdmin/new_DefaultTextsAdmin/pages/Detail.tsx [hunk 20]
```diff
@@ -0,0 +1,52 @@
...
+        BackLink={<BackLink to={redirectURL}>Terug naar overzicht</BackLink>}
+      />
+      {/*{isLoading && <LoadingIndicator />}*/}
+
+      <form
+        data-testid="new-default-text-admin-form"
+        className="new-default-text-admin_form"
+      >
+        <FormContainer data-testid="new-default-text-admin-formContainer">
+          <div>
```
[the remark this answers is not in the document]
Comment (SireeKoolenWijkstra) [reply src/signals/incident-management/containers/DefaultTextsAdmin/new_DefaultTextsAdmin/pages/Detail.tsx]: Yup, completely

File: src/shared/services/configuration/endpoint-definitions.js (hunk 8 above)
Comment (SireeKoolenWijkstra) [reply src/shared/services/configuration/endpoint-definitions.js:99]: This is old, right?

Comment (SireeKoolenWijkstra) [reply src/shared/services/configuration/endpoint-definitions.js:99]: added info about the patch and delete

File: src/signals/incident-management/containers/StandardTextsAdmin/components/DetailPage/Detail.tsx (hunk 9 above)
Comment (SireeKoolenWijkstra) [reply src/signals/incident-management/containers/StandardTextsAdmin/components/DetailPage/Detail.tsx:40]: Did it, done it.

File: src/signals/incident-management/containers/StandardTextsAdmin/components/DetailPage/utils.ts (hunk 10 above)
Comment (SireeKoolenWijkstra) [reply src/signals/incident-management/containers/StandardTextsAdmin/components/DetailPage/utils.ts:8]: Done

File: src/signals/incident-management/containers/StandardTextsAdmin/components/SelectedSubcategories/SelectedSubcategories.tsx (hunk 11 above)
Comment (SireeKoolenWijkstra) [reply src/signals/incident-management/containers/StandardTextsAdmin/components/SelectedSubcategories/SelectedSubcategories.tsx:11]: Added it to the stylesheet. There is a generic component ErrorMessage that's now directly referenced.

File: src/signals/incident-management/containers/StandardTextsAdmin/components/SelectedSubcategories/mock-subcategories.ts (hunk 12 above)
Comment (SireeKoolenWijkstra) [reply src/signals/incident-management/containers/StandardTextsAdmin/components/SelectedSubcategories/mock-subcategories.ts:3]: Did it, thought we wanted to move away from the _test_folders.

File: src/signals/incident-management/containers/StandardTextsAdmin/components/Summary/Summary.tsx (hunk 13 above)
Comment (SireeKoolenWijkstra) [reply src/signals/incident-management/containers/StandardTextsAdmin/components/Summary/Summary.tsx:27]: removed it

File: src/signals/incident-management/index.js (hunk 14 above)
Comment (SireeKoolenWijkstra) [reply src/signals/incident-management/index.js:117]: @OscarBakker If it is not the correct way, how do I then do the BackLink? Cause I ran into trouble there and this was my solution. (all right stackoverflow's solution)

File: src/signals/incident-management/containers/StandardTextsAdmin/components/DetailPage/Detail.test.tsx (hunk 15 above)
Comment (SireeKoolenWijkstra) [reply src/signals/incident-management/containers/StandardTextsAdmin/components/DetailPage/Detail.test.tsx:46]: Added space just about everywhere, readability is a very nice to have

File: internals/testing/msw-server.ts (hunk 17 above)
Comment (SireeKoolenWijkstra) [reply internals/testing/msw-server.ts:237]: Changed it

File: internals/testing/msw-server.ts (hunk 18 above)
Comment (SireeKoolenWijkstra) [reply internals/testing/msw-server.ts:264]: Changed it

File: src/signals/incident-management/containers/StandardTextsAdmin/components/DetailPage/Detail.test.tsx (hunk 19 above)
Comment (SireeKoolenWijkstra) [reply src/signals/incident-management/containers/StandardTextsAdmin/components/DetailPage/Detail.test.tsx:20]: Removed it

**Review by vdegraaf — COMMENTED** [review]
Looks good! Two points:

- I'd expect the cursor to be a pointer in the overviewpage. Has been commented twice already.
- When i don't change anything and click the "opslaan" button, it doesn't do anything. In other parts of the application you do return to the overview page then.

File: src/shared/services/configuration/endpoint-definitions.js [hunk 21]
```diff
@@ -94,7 +94,7 @@ export default {
     '/signals/v1/private/status-messages/category/',
   // Private. Used by the incident management module to retrieve and query standard texts.
   STANDARD_TEXTS_ENDPOINT: '/signals/v1/private/status-messages/',
-  // Private. Used by the incident management module to retrieve and search standard texts.
+  // Private. Used by the incident management module to retrieve and search standard texts, to patch updated data or to delete a standard text..
```
[the remark this answers is not in the document]
Comment (OscarBakker) [reply src/shared/services/configuration/endpoint-definitions.js:97]: this is just a get

File: src/signals/incident-management/containers/StandardTextsAdmin/components/DetailPage/Detail.tsx [hunk 22]
```diff
@@ -0,0 +1,280 @@
...
+    resolver: yupResolver(schema),
+    defaultValues: { ...defaultValues },
+  })
+
+  const { handleSubmit, formState, reset, getValues } = formMethods
+
+  const onSubmit = useCallback(() => {
+    const hasDirtyFields = Object.keys(formState.dirtyFields).length > 0
+
+    hasDirtyFields &&
```
Comment (OscarBakker) [inline src/signals/incident-management/containers/StandardTextsAdmin/components/DetailPage/Detail.tsx:95]: could also just reverse this logic. Then one check 'has not, navigate and return'

File: src/signals/incident-management/containers/StandardTextsAdmin/components/DetailPage/Detail.tsx [hunk 23]
```diff
@@ -0,0 +1,280 @@
...
+  useEffect(() => {
+    // Prefill form with data from query
+    defaultValues && reset(defaultValues)
+  }, [defaultValues, reset])
+
+  useEffect(() => {
+    if (isSuccess) {
+      setWaitForTimeout(true)
+
+      // Set delay to wait for search endpoint to be updated
```
Comment (OscarBakker) [inline src/signals/incident-management/containers/StandardTextsAdmin/components/DetailPage/Detail.tsx:128]: aai waiting with a timeout, hope we can remove this one in a following story

File: src/signals/incident-management/containers/StandardTextsAdmin/components/PageHeader/PageHeader.tsx [hunk 24]
```diff
@@ -0,0 +1,33 @@
...
+
+import { StyledHeading, StyledHeadingWrapper, StyledSection } from './styled'
+
+type Props = {
+  title: string
+  children?: ReactNode
+  backLink?: ReactNode
+  filter?: {
+    name: string
+    options: Option
```
Comment (OscarBakker) [inline src/signals/incident-management/containers/StandardTextsAdmin/components/PageHeader/PageHeader.tsx:15]: Option[] ?

File: src/signals/incident-management/containers/StandardTextsAdmin/components/SelectedSubcategories/SelectedSubcategories.tsx [hunk 25]
```diff
@@ -0,0 +1,49 @@
...
+import { useSelector } from 'react-redux'
+
+import Button from 'components/Button'
+import { makeSelectSubCategories } from 'models/categories/selectors'
+
+import { StyledErrorMessage } from './styled'
+
+export interface Props {
+  name: string
+  error?: FieldError
```
Comment (OscarBakker) [inline src/signals/incident-management/containers/StandardTextsAdmin/components/SelectedSubcategories/SelectedSubcategories.tsx:15]: optional should be last

File: src/signals/incident-management/containers/StandardTextsAdmin/components/SelectedSubcategories/SelectedSubcategories.tsx [hunk 26]
```diff
@@ -0,0 +1,49 @@
...
+    return value
+      ?.map((id) => {
+        return subcategories?.find((sub) => {
+          return sub.fk == id
+        })
+      })
+      .map((cat) => cat?.public_name ?? cat?.name)
+      .filter(Boolean)
+      .join(', ')
+  }, [subcategories, value])
```
Comment (OscarBakker) [inline src/signals/incident-management/containers/StandardTextsAdmin/components/SelectedSubcategories/SelectedSubcategories.tsx:35]: maybe better: return value?.map(id => {
  const sub = subcategories?.find(sub => sub.fk == id);
  return sub?.public_name ?? sub?.name;
}).filter(Boolean).join(', ');

**Review by vdegraaf — APPROVED** [review]

File: internals/testing/msw-server.ts [hunk 27]
```diff
@@ -254,11 +255,19 @@ const handlers = [
     res(ctx.status(200), ctx.json(statusMessageTemplatesFixture))
   ),
 
+  rest.get(API.STANDARD_TEXTS_DETAILS_ENDPOINT, (_req, res, ctx) =>
+    res(ctx.status(200), ctx.json(detail))
+  ),
+
   // PATCH
```
[the remark this answers is not in the document]
Comment (SireeKoolenWijkstra) [reply internals/testing/msw-server.ts:262]: Done

File: internals/testing/msw-server.ts [hunk 28]
```diff
@@ -280,6 +289,11 @@ const handlers = [
     res(ctx.status(200))
   ),
 
+  //DELETE
```
[the remark this answers is not in the document]
Comment (SireeKoolenWijkstra) [reply internals/testing/msw-server.ts:292]: Done

File: src/shared/services/configuration/endpoint-definitions.js (hunk 21 above)
[the remark this answers is not in the document]
Comment (SireeKoolenWijkstra) [reply src/shared/services/configuration/endpoint-definitions.js:97]: I use this endpoint now also to patch and delete.

File: src/signals/incident-management/containers/StandardTextsAdmin/components/SelectedSubcategories/SelectedSubcategories.tsx (hunk 25 above)
Comment (SireeKoolenWijkstra) [reply src/signals/incident-management/containers/StandardTextsAdmin/components/SelectedSubcategories/SelectedSubcategories.tsx:15]: ok

**Pull request merged**